pkg/api/testing/fuzzer.go

```diff
@@ -190,21 +189,6 @@ func FuzzerFor(t *testing.T, version unversioned.GroupVersion, src rand.Source)
 				*j = t
 			}
 		},
-		func(pb map[docker.Port][]docker.PortBinding, c fuzz.Continue) {
```

Comment:
These code should've been removed ages ago! 

We used to have docker.Container type exposed in the k8s pod object, so we had to fuzz the these fields for testing api objects. We no longer have raw docker types in our api anymore.